cmake/ProjectCryptopp.cmake

```diff
@@ -140,8 +140,8 @@ ExternalProject_Add(cryptopp
         -DCMAKE_POSITION_INDEPENDENT_CODE=${BUILD_SHARED_LIBS}
         -DBUILD_SHARED=Off
         -DBUILD_TESTING=Off
-        -DCMAKE_C_FLAGS=-Wa,-march=generic64
-        -DCMAKE_CXX_FLAGS=-Wa,-march=generic64
+        # -DCMAKE_C_FLAGS=-Wa,-march=generic64
```

Comment:
i think group signature may use this, maybe use had check it run correctly ?😀

Reply:
We don't have group sign yet. If we have and we need, then recover it.